indexer/LlvmAdapter.h

```diff
@@ -19,6 +23,9 @@ template <typename T> std::string format(const T &llvmValue) {
   return buffer;
 }
 
+// Meant to only be used for debugging purposes
```

Comment:
Logs also fall into the "for debugging purposes", and it's fine to use this in logs. However, the log level is controlled dynamically instead of statically.